core/src/main/java/ai/timefold/solver/core/impl/solver/scope/SolverScope.java

```diff
@@ -355,4 +381,17 @@ public void destroyYielding() {
         }
     }
 
+    public void incrementMoveEvaluationCountPerType(Move<?> move) {
+        addMoveEvaluationCountPerType(move.getSimpleMoveTypeDescription(), 1L);
+    }
+
+    public void addMoveEvaluationCountPerType(String moveType, long count) {
```

Comment:
Why is this method public, if we already have the one above?

Reply:
We use it in the MT synchronization process.